### PR TITLE
Update webrtc

### PR DESCRIPTION
**Pull request by philippewarren**

### CODE REVIEW

**Comment by doumdi** [comment]
@philippewarren Are we ready to merge this branch ?

**Comment by philippewarren** [comment]
@doumdi Maybe we could add something in the README about the Windows/Ubuntu 18.04 support (not supported, use the older version) before merging.

File: .github/workflows/build_only.yml [hunk 1]
```diff
@@ -15,50 +15,50 @@ jobs:
```
Comment (mamaheux) [inline .github/workflows/build_only.yml:15]: Remove arm32

File: .github/workflows/build_only.yml [hunk 2]
```diff
@@ -68,43 +68,42 @@ jobs:
```
Comment (mamaheux) [inline .github/workflows/build_only.yml:68]: Use macos-11 and macos-12.

**Review by mamaheux — APPROVED** [review]

**Review by doumdi — APPROVED** [review]

**Pull request merged**